ruby.rb

```diff
@@ -0,0 +1,10 @@
+class BadFlog
+  def some_method
+    super
+  rescue StandardError => error
+
+    if this(&wat=foo) == true
+      that = "something"
```

Comment:
Useless assignment to variable - that.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.